diff --git a/arch/arm/boot/dts/qcom/kenzo/msm8956-kenzo.dtsi b/arch/arm/boot/dts/qcom/kenzo/msm8956-kenzo.dtsi
index 8f6133d..49c79a2 100644
--- a/arch/arm/boot/dts/qcom/kenzo/msm8956-kenzo.dtsi
+++ b/arch/arm/boot/dts/qcom/kenzo/msm8956-kenzo.dtsi
@@ -79,7 +79,7 @@
 };
 
 &i2c_6 {
-	awinic@45 { /* Breath Light-Driven */
+	aw2013@45 { /* Breath Light-Driven */
 		compatible = "awinic,aw2013";
 		reg = <0x45>;
 		vdd-supply = <&pm8950_l10>;
@@ -89,7 +89,7 @@
 			aw2013,name = "red";
 			aw2013,id = <0>;
 			aw2013,max-brightness = <255>;
-			aw2013,max-current = <1>;
+			aw2013,max-current = <3>;
 			aw2013,rise-time-ms = <2>;
 			aw2013,hold-time-ms = <1>;
 			aw2013,fall-time-ms = <2>;
@@ -99,7 +99,7 @@
 			aw2013,name = "green";
 			aw2013,id = <1>;
 			aw2013,max-brightness = <255>;
-			aw2013,max-current = <1>;
+			aw2013,max-current = <3>;
 			aw2013,rise-time-ms = <2>;
 			aw2013,hold-time-ms = <1>;
 			aw2013,fall-time-ms = <2>;
@@ -109,7 +109,7 @@
 			aw2013,name = "blue";
 			aw2013,id = <2>;
 			aw2013,max-brightness = <255>;
-			aw2013,max-current = <1>;
+			aw2013,max-current = <3>;
 			aw2013,rise-time-ms = <2>;
 			aw2013,hold-time-ms = <1>;
 			aw2013,fall-time-ms = <2>;
